katello/hooks/pre/30-upgrade.rb

```diff
@@ -91,6 +102,7 @@ def fail_and_exit(message)
   if katello
     upgrade_step :migrate_candlepin, :run_always => true
     upgrade_step :migrate_foreman, :run_always => true
+    upgrade_step :postgresql_10_upgrade
```

Comment:
Can you move this to line 95 so we perform the DB migration before other migrations?

Reply:
See https://github.com/theforeman/foreman-installer/pull/407#issue-343739127

Reply:
Why is that needed? Regardless, it needs the same if conditions (and `if katello` isn't needed anymore then). Otherwise you break remote database support.